### PR TITLE
New: Update BaseViewer to use box-annotations npm package

### PR DESCRIPTION
**Pull request by pramodsum**

### CODE REVIEW

File: src/lib/viewers/BaseViewer.js [hunk 1]
```diff
@@ -661,16 +657,14 @@ class BaseViewer extends EventEmitter {
             return;
         }
 
-        /* global BoxAnnotations */
         const boxAnnotations = new BoxAnnotations();
```
Comment (JustinHoldstock) [inline src/lib/viewers/BaseViewer.js]: IMO You might want a try/catch around the instancing line. If BoxAnnotations doesn't exist, it'll blow up, potentially stopping preview from working

Comment (pramodsum) [reply src/lib/viewers/BaseViewer.js]: 👍

**Comment by pramodsum** [comment]
Removing annotations code in a separate commit https://github.com/box/box-content-preview/pull/451

File: build/webpack.common.config.js [hunk 2]
```diff
@@ -50,15 +50,15 @@ module.exports = (language) => {
                     }),
                     exclude: [
                         path.resolve('src/third-party'),
-                        path.resolve('node_modules')
+                        path.resolve('^(?!node_modules/box-annotations).*')
```
Comment (pramodsum) [inline build/webpack.common.config.js]: This excludes all paths in `node_modules` except `box-annotations`

**Pull request merged**